src/lib/geo-location.js

```diff
@@ -56,10 +54,18 @@ const getGeoLocation = async ({ mode, offline, state }) => {
     }
   }
 
-  // If the `--geo` option is set to `mock`, we use the mock location. Also,
-  // if the `--offline` option was used, we can't talk to the API, so let's
-  // also use the mock location.
-  if (mode === 'mock' || offline) {
+  // If the `--geo` option is set to `mock`, we use the default mock location.
+  // If the `--offline` option was used, we can't talk to the API, so let's
+  // also use the mock location.  Otherwise, use the country code passed in by
+  // the user.
+  if (mode === 'mock' || offline || geoCountry) {
+    if (geoCountry) {
+      return {
+        city: 'Netlitown',
+        country: { code: geoCountry, name: 'Mock Country' },
+        subdivision: { code: 'JS', name: 'Jamstackvania' },
```

Comment:
For the sake of clarity, especially for folks who learned / are learning English as a 2nd+ language, and to avoid potential translation troubles, I suggest avoiding fun pun names like "Netlitown" and "Jamstackvania" in favor of explicit placeholder names like "Fake city" or "Mock city" / "Fake subdivision" or "Mock subdivision"